inflation/lp/InflationLP.py

```diff
@@ -0,0 +1,1890 @@
+            default ``True``.
+        include_all_outcomes : bool, optional
+            Whether to include all outcomes in the LP, even if they are not
+            used in the scenario. By default ``False``.
+        verbose : int, optional
+            Optional parameter for level of verbose:
+                * 0: quiet (default),
+                * 1: monitor level: track program process and show warnings,
+                * 2: debug level: show properties of objects created.
+        """
+        self.problem_type = "lp"
+        self.supports_problem = supports_problem
+        if verbose is not None:
+            if inflationproblem.verbose > verbose:
+                warn("Overriding the verbosity from InflationProblem")
+            self.verbose = verbose
+        else:
+            self.verbose = inflationproblem.verbose
+        self.nonfanout = nonfanout
+        self.default_non_negative = default_non_negative
+        if self.verbose > 1:
+            print(inflationproblem)
+
+        # Inheritance
+        self.InflationProblem = inflationproblem
+        self.names = inflationproblem.names
+        self.names_to_ints = inflationproblem.names_to_ints
+        self._lexrepr_to_names = inflationproblem._lexrepr_to_names
+        self.nr_sources = inflationproblem.nr_sources
+        self.nr_parties = inflationproblem.nr_parties
+        self.hypergraph = inflationproblem.hypergraph
+        self.network_scenario = inflationproblem.is_network
+        self.inflation_levels = inflationproblem.inflation_level_per_source
+        self.setting_cardinalities = inflationproblem.settings_per_party
+        self.outcome_cardinalities = inflationproblem.outcomes_per_party
+        self.private_setting_cardinalities = inflationproblem.private_settings_per_party
+        self.expected_distro_shape = inflationproblem.expected_distro_shape
+        self.rectify_fake_setting = inflationproblem.rectify_fake_setting
+        self.factorize_monomial_1d = inflationproblem.factorize_monomial_1d
+        self._is_knowable_q_non_networks = \
+            inflationproblem._is_knowable_q_non_networks
+        self._nr_properties = inflationproblem._nr_properties
+        self.np_dtype = inflationproblem._np_dtype
+        self._lexorder = inflationproblem._lexorder
+        self._nr_operators = inflationproblem._nr_operators
+        self._lexrange = np.arange(self._nr_operators)
+        self.lexorder_symmetries = inflationproblem.lexorder_symmetries
+        self._from_2dndarray = inflationproblem._from_2dndarray
+        self.mon_to_lexrepr = inflationproblem.mon_to_lexrepr
+        self.blank_bool_vec = np.zeros(self._nr_operators, dtype=bool)
+        self._ortho_groups_per_party = inflationproblem._ortho_groups_per_party
+        self.has_children = inflationproblem.has_children.copy()
+        if include_all_outcomes or supports_problem: # HACK to fix detection of incompatible supports. (Can be fixed upon adding set_extra_equalities)
```

Comment:
Has this been fixed?

Reply:
Openes a new issue to keep track of this.